src/components/FocusTrap/WIDE_LAYOUT_INACTIVE_SCREENS.ts

```diff
@@ -0,0 +1,34 @@
+import NAVIGATORS from '@src/NAVIGATORS';
+import SCREENS from '@src/SCREENS';
+
+const WIDE_LAYOUT_INACTIVE_SCREENS: string[] = [
```

Comment:
I'm confused about what this file is and why it's needed. Can we:

- at minimum, add a comment to explain what this is and why it's needed
- (ideally, if possible) derive it so that it doesn't need to be manually updated

Reply:
Comment added